rel/ballcolor.cpp

```diff
@@ -14,56 +14,22 @@
 
 namespace ballcolor {
 
-static s32 ballColorID = 0;
```

Comment:
Most of the conversations were resolved, did you forget to commit the rest of your changes?